src/web/mjs/connectors/MangaDex.mjs

```diff
@@ -11,7 +11,17 @@ export default class MangaDex extends Connector {
         this.url = 'https://mangadex.org';
         this.api = 'https://api.mangadex.org';
         this.requestOptions.headers.set('x-referer', this.url);
-        this.throttleGlobal = 100;
+        this.requestOptions.headers.set('x-sec-ch-ua', '" Not A;Brand";v="99", "Chromium";v="96", "Google Chrome";v="96"');
```

Comment:
What is the purpose of this header? It is very unlikely that any `sec-*` headers have an impact on requests.

Reply:
that is the header that stops the captcha-v3 404 error and the 503 errors

i went through every header different from a chrome request and HN request this was the one that bade the differance